src/mocks/data/utils.ts

```diff
@@ -80,3 +80,9 @@ export const createTextFileItem = (item: any): TextFileResponseModelBaseModel =>
 	name: item.name,
 	content: item.content,
 });
+
+export const createItem = (item: any): any => ({
```

Comment:
This is a bit too lose name in my opinion. What type of item is this?

Plus I don't seem to spot where it begin used? so could you confirm what purpose it has? :-) Thanks

Reply:
So this item mathches this interface https://github.com/umbraco/Umbraco.CMS.Backoffice/blob/dd443a58013869713047d6cc8d594677588f6658/src/external/backend-api/src/models/ScriptItemResponseModel.ts#L8
Since I am not really sure what type of item this is I am renaming it to the server model